gui/default/syncthing/core/syncthingController.js

```diff
@@ -378,6 +382,8 @@ angular.module('syncthing.core')
             $scope.config.options._globalAnnounceServersStr = $scope.config.options.globalAnnounceServers.join(', ');
             $scope.config.options._urAcceptedStr = "" + $scope.config.options.urAccepted;
 
+            $scope.config.gui["showRemoteGUI"] = $scope.showRemoteGUI;
```

Comment:
This is an artificial property, so doing what we do with _urAcceptedStr  et all, prefixing it with an underscore would be nice.
Could the way we assign things be normalised too?

Or I guess the name of the advanced config is driven from this hence it has to be this way?

Reply:
Yeah, this is seen by the user, so it make more sense to keep as is. If there exist a way to have a name different from what is displayed, I do not know it.